packages/main/src/themes/Menu.css

```diff
@@ -116,21 +53,4 @@
 .ui5-menu-rp[sub-menu][actual-placement-type="Left"] {
 	margin-top: 0.25rem;
 	margin-inline: var(--_ui5_menu_submenu_placement_type_left_margin_offset);
-}
-
-.ui5-menu-item::part(additional-text) {
-	margin-inline-start: var(--_ui5_menu_item_additional_text_start_margin);
-	color: var(--sapContent_LabelColor);
-	min-width: max-content;
-}
-
-.ui5-menu-item-text {
```

Comment:
I think this should stay because even if the max-width of the menu is removed, it is still possible to have longer text in the menu item.

Reply:
Also, sometimes there is no margin in front of additional text container, please check.

Reply:
Done